templates/paloalto_panos_show_interface_management.textfsm

```diff
@@ -0,0 +1,18 @@
+Value Name (\S+)
+Value MACaddress ([a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2})
+Value IPv4 (\S+)
+Value IPv4Netmask (\S+)
+Value IPv4Gateway (\S+)
+Value IPv6 (\S+)
+Value IPv6LinkLocal (\S+)
+Value IPv6Gateway (\S+)
+
+Start
+  ^Name: ${Name}
+  ^\s+Port MAC address ${MACaddress}
+  ^Ip address: ${IPv4}
+  ^Netmask: ${IPv4Netmask}
```

Comment:
Please change all the static spacing to use either `\s*` or `\s+`. Most of them should be `\s+` since it's expecting at least one space.